packages/synapse-react-client/src/components/CertificationQuiz/CertificationQuiz.tsx

```diff
@@ -94,135 +126,189 @@ const CertificationQuiz: React.FunctionComponent = () => {
+    <div>
+      {passingRecord && !isTakingQuiz && (
+        <>
+          {(!passingRecord.passed || !isCertified) && (
+            <Alert severity="error">
+              {!passingRecord.passed && <AlertTitle>Quiz Failed</AlertTitle>}
+              <Typography variant="body1" sx={{ marginTop: '5px' }}>
+                Please review the results from the previous attempt, and{' '}
                 <Link
                   href="#"
                   onClick={e => {
                     e.preventDefault()
                     handleRetakeQuiz()
                   }}
                 >
-                  try again
+                  retake the quiz
                 </Link>
                 .
               </Typography>
-            )}
-          </>
-        </div>
+            </Alert>
+          )}
+          {passingRecord.passed && isCertified && (
+            <Alert severity="success">
+              <AlertTitle>Quiz Passed</AlertTitle>
+              <Typography variant="body1" sx={{ marginTop: '5px' }}>
+                {`You passed the Synapse Certification Quiz on ${formatDate(
+                  dayjs(passingRecord.passedOn),
+                )}`}
+              </Typography>
+            </Alert>
+          )}
+          <Typography
+            variant="sectionTitle"
+            sx={{ marginTop: '20px', marginBottom: '20px' }}
+          >
+            Score: {passingRecord.score} / {quiz?.questions.length}
+          </Typography>
+        </>
       )}
       <div className="CertificationQuiz__container">
-        <Button
-          onClick={() => window.open(GETTING_STARTED_URL, '_blank')}
-          className="help-button"
-          color="secondary"
-          variant="contained"
-        >
-          <HelpOutlineTwoTone
-            className="HelpButton"
-            style={{ marginRight: '4px' }}
-          />
-          Help
-        </Button>
-        {quiz ? (
-          <div dangerouslySetInnerHTML={{ __html: quiz.header }}></div>
-        ) : (
-          ''
+        {quiz && isTakingQuiz && (
+          <>
+            <Button
+              onClick={() => window.open(GETTING_STARTED_URL, '_blank')}
+              className="help-button"
+              color="secondary"
+              variant="contained"
+            >
+              <HelpOutlineTwoTone
+                className="HelpButton"
+                style={{ marginRight: '4px' }}
+              />
+              Help
+            </Button>
+            <div dangerouslySetInnerHTML={{ __html: quiz.header }}></div>
+            <form ref={formRef} onSubmit={e => e.preventDefault()}>
+              <ol>
+                {quiz?.questions.map(question => {
+                  const isHelp = question.helpText || question.docLink
+                  return (
+                    <li
+                      key={question.questionIndex}
+                      role={question.exclusive ? 'radiogroup' : undefined}
+                    >
+                      <div
```

Comment:
Initially used mui Typography, but this div dangerouslySetInnerHTML meant the paragraph had a div child, which I don't like.  So I left it.